examples/flatmap/flatmap_slice_test.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"github.com/reactivex/rxgo/observable"
+	"github.com/reactivex/rxgo/observer"
+	"testing"
```

Comment:
gofmt run pls

Internal library is imported before

Reply:
Strange thing is that my gofmt puts "testing" library on the bottom of imports. I'll correct this manually.